src/js/modules/articles/components/Dateline.js

```diff
@@ -13,7 +14,9 @@ const styles = {
 
 // TODO: format dates
 const Dateline = ({ classes, article }) => {
-  return <p className={classes.Dateline}>7:20 A.M.</p>;
+  return (<p className={classes.Dateline}>
+  	{formatDate(article.createdAt)}
+  </p>);
```

Comment:
Coding style: do not have any code on the same line as the `return` parentheses.